python/podio/root_io.py

```diff
@@ -3,11 +3,17 @@
 
 from ROOT import gSystem
 
+from utils import convert_to_str_paths
+
 gSystem.Load("libpodioRootIO")  # noqa: E402
 from ROOT import podio  # noqa: E402 # pylint: disable=wrong-import-position
 
-from podio.base_reader import BaseReaderMixin  # pylint: disable=wrong-import-position # noqa: E402
-from podio.base_writer import BaseWriterMixin  # pylint: disable=wrong-import-position # noqa: E402
+from podio.base_reader import (  # pylint: disable=wrong-import-position # noqa: E402
+    BaseReaderMixin,
+)
+from podio.base_writer import (  # pylint: disable=wrong-import-position # noqa: E402
+    BaseWriterMixin,
+)
```

Comment:
```suggestion
from podio.base_reader import BaseReaderMixin  # pylint: disable=wrong-import-position # noqa: E402
from podio.base_writer import BaseWriterMixin  # pylint: disable=wrong-import-position # noqa: E402
```
No unrelated format changes please